tools/ResxGen/PowerShell-Core-Instrumentation.man

```diff
@@ -0,0 +1,5635 @@
+      <provider
+          guid="{f90714a8-5509-434a-bf6d-b1624c8a19a2}"
+          messageFileName="System.Management.Automation.dll"
+          name="PowerShellCore"
+          resourceFileName="System.Management.Automation.dll"
+          symbol="PS_PROVIDER"
+          >
+        <!--The following section defines the events which can be written
+            from PowerShell-->
+        <events>
+          <!--Here are the guidelines on assigning values to events:
+              1. Reserved      - 0x0001 - 0x1000
+              2. Commands      - 0x1001 - 0x2000
+              3. Runspace      - 0x2001 - 0x3000
+              4. Pipeline      - 0x3001 - 0x4000
+              5. Session       - 0x4001 - 0x5000
+              6. Protocol      - 0x5001 - 0x6000
+              7. Host          - 0x6001 - 0x7000
+              8. Serialization - 0x7001 - 0x8000
+              9. Transport     - 0x8001 - 0x9000
+             10. Plugin        - 0x9001 - 0xA000
+             11. PerfTrack     - 0xA001 - 0xB000
+             12. M3P           - 0xB001 - 0xC000
+             13. Debug Events  - 0xC001 - 0xD000
+             14. ScheduledJob  - 0xD001 - 0xD0FF
+             15. NamedPipeIPC  - 0xD100 - 0xE000
+
+             Values for events in operational channel will be in the range
+             0xX001 - 0xXF00 and for those in the analytic channel will be
+             in the range from 0xXF01 - 0xXFFF
+
+             The version should be 1 for all events defined in the V2
+             timeframe
+             -->
+          <!--ScheduledJob events -->
+          <event
+              channel="C_OPERATIONAL"
+              level="win:Informational"
+              message="$(string.PS_PROVIDER.event.E_O_ScheduledJobStarted.message)"
+              opcode="Method"
+              symbol="ScheduledJobStarted"
+              task="ScheduledJob"
+              template="T_ScheduledJobStarted"
+              value="0xD001"
+              version="1"
+              />
+          <event
+              channel="C_OPERATIONAL"
+              level="win:Informational"
+              message="$(string.PS_PROVIDER.event.E_O_ScheduledJobCompleted.message)"
+              opcode="Method"
+              symbol="ScheduledJobCompleted"
+              task="ScheduledJob"
+              template="T_ScheduledJobCompleted"
+              value="0xD002"
+              version="1"
+              />
+          <event
+              channel="C_OPERATIONAL"
+              level="win:Error"
+              message="$(string.PS_PROVIDER.event.E_O_ScheduledJobException.message)"
+              opcode="Exception"
+              symbol="ScheduledJobError"
+              task="ScheduledJob"
+              template="T_ScheduledJobError"
+              value="0xD003"
+              version="1"
+              />
+          <!--M3P events-->
+          <event
+              channel="C_ANALYTIC"
+              keywords="PSWorkflow"
```

Comment:
I assume this is copied directly from Windows PowerShell manifest.  I see that it has "PSWorkflow" keywords.  Since we don't support workflows they can be removed at some point.

Reply:
Yes, we'll need to scrub all entries that are no longer referenced.

Reply:
Please do the scrub before this is merged - no sense in taking something that never should have been there in the first place.

Reply:
I'll do the scrub as part of the Windows manifest work; due next.